test/JDBC/input/BABEL-4271-vu-prepare.sql

```diff
@@ -0,0 +1,8 @@
+-- Test to check like escape null and like escape ''
+create table testvikasprj(a varchar(30), b varchar(30));
+go
+
+insert into testvikasprj values ('cbc','[c-a]bc');
```

Comment:
Let us have more edge case. One of them can be something like:
insert into t1 values('abc',' abc")
The difference between the inputs is, the second column has an extra space before abc. So, think of cases like that. And make sure we are getting expected results.

Reply:
Added